CMakeLists.txt

```diff
@@ -88,7 +89,7 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU"
     endif ()
 endif ()
 
-
+include_directories(${R_INCLUDE_DIR} SYSTEM)
```

Comment:
Je pense qu'il faut plutot utiliser `target_include_directories` pour la target qui doit inclure ces headers, plutot que des instructions globales.

Reply:
Actually since xeus-r will link with R, adding `{R_LIBRARIES}` to the `target_link_libraries` [arguments](https://github.com/jupyter-xeus/xeus-r/blob/main/CMakeLists.txt#L205) should be enough

Reply:
Thanks. I'll keep experimenting with this